index.html

```diff
@@ -45,9 +48,196 @@ <h2>Software Developer</h2>
+                    </div>
+
+                </div>
+                <div class="hr-flex">
+                    <hr>
+                </div>
+                <div class="background-img"></div>
 
+                <div class="social-icon">
+                    <ul>
+                        <li><img src="./asset/Vector1.png" alt=""></li>
+                        <li><img src="./asset/‌linkedin.png" alt=""></li>
+                        <li><img src="./asset/Vectorthumb.png" alt=""></li>
+                        <li><img src="./asset/Vectortwitter.png" alt=""></li>
+                        <li><img src="./asset/Vectorm.png" alt=""></li>
+                    </ul>
+                </div>
+
+                <div>
+                    <div class="work-icon">
+                        <div><img src="./asset/Group_55.png" alt=""></div>
+                        <div>
+                            <p>work</p>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+            <section>
+                <div class="work-profile">
+                    <h3>MY RECENT WORK</h3>
+                </div>
+                <div class="hr-flex">
+                    <hr>
+                </div>
+                <div class="work-post-background">
+                 
+                </div>
+                <div class="work-container">
+                    
+                    <div class="grid">
+                        <div class="placeholder spc-style">
+               
+                        </div>
+                        <div class="work-post">
+                            <div class="work-name">
+                                <h3>Multi-Post Stories</h3>
+                            </div>
+                            <div class="work-desc">
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                                    been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                                    took a standard dummy text.</p>
+                            </div>
+                        </div>
+                        <div class="tools">
+                            <div class="tool-skill">
+                                <ul>
+                                    <li><a href=""></a>css</li>
+                                    <li><a href=""></a>html</li>
+                                    <li><a href=""></a>bootstrap</li>
+                                    <li><a href=""></a>Ruby</li>
+                                </ul>
+                            </div>
+    
+                        </div>
+                        <div class="project-btn">
+                            <button type="button" class="btn">see project</button>
+                        </div>
+                        <div class="placeholder spc-style">
+    
+                        </div>
```

Comment:
- [ optional ] It will be nice to wrap the content of each project in a container and use flexbox to display the content.

Reply:
thanks i was advised to use grid on this but I would like at how I flex could be handy, flex was used in the list items

Reply:
You are right @xtrahuman. It is required to use a grid to display the projects.